Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -1104,34 +1102,30 @@ describe(
     });
 
     it("verify memory usage statistics for shared resources", function () {
+      ResourceCache.statistics.clear();
       // Six tiles total:
       // * Two b3dm tiles - no shared resources
       // * Two i3dm tiles with embedded glTF - no shared resources
       // * Two i3dm tiles with external glTF - shared resources
       // Expect to see some saving with memory usage since two of the tiles share resources
       // All tiles reference the same external texture but texture caching is not supported yet
       // TODO : tweak test when #5051 is in
```

Comment:
Given that #5051 was never merged and statistics work different, so maybe this line should be removed?